### PR TITLE
Add new Geography definitions for International and no-Geography

### PR DESCRIPTION
**Pull request by diversemix**
# Description

- If geo is blank it defaults to no geography 
- Adds definition for no geography
- Removes old and adds new definition for international

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change

## How Has This Been Tested?

Please describe the tests that you added to verify your changes.

## Reviewer Checklist

- [ ] The PR represents a single feature (small driveby fixes are also ok)
- [ ] The PR includes tests that are sufficient for the level of risk
- [ ] The code is sufficiently commented, particularly in hard-to-understand areas
- [ ] Any required documentation updates have been made
- [ ] Any TODOs added are captured in future tickets
- [ ] No FIXMEs remain

### CODE REVIEW

**Comment by diversemix** [comment]
@joel-wright Do we want a test for ingesting a document with a blank geo?

**Comment by diversemix** [comment]
Also just realised that the "validation" operation also mutates the data (geo) which is a side-effect we may not wish to have :cry:

File: app/data_migrations/populate_geography.py [hunk 1]
```diff
@@ -5,9 +5,12 @@
 from sqlalchemy.orm import Session
 
 from app.db.models.law_policy import Geography
+from app.db.models.law_policy.geography import GeoStatistics
 
 from .utils import has_rows, load_tree
 
+CPR_DEFINED_GEOS = {"XXA": "No Geography", "XXB": "International"}
```
Comment (joel-wright) [inline app/data_migrations/populate_geography.py:12]:
```suggestion
CPR_DEFINED_GEOS = {"XAA": "No Geography", "XAB": "International"}
```

Comment (diversemix) [reply app/data_migrations/populate_geography.py:12]: Let me make consts so we can use them in the code

**Review by joel-wright — COMMENTED** [review]
The `INT` code also needs to be removed from the csv & json data files

**Comment by diversemix** [comment]
Needs to also do the subst for a blank for geography in the ingest stage.

**Review by joel-wright — APPROVED** [review]

**Pull request merged**